docs/api/snowflake/vector-data/Function.md

```diff
@@ -2285,6 +2285,33 @@ Result:
 +---------------------------------------------------------------+
 ```
 
+## ST_Rotate
+
+Introduction: Rotates a geometry by a specified angle in radians counter-clockwise around a given origin point. The origin for rotation can be specified as either a POINT geometry or x and y coordinates. If the origin is not specified, the geometry is rotated around POINT(0 0).
+
+!!!note
+    This function retains the z-index for 3D geometries and is compatible with Circular Strings, Curves, Polyhedral surfaces, Triangles, and Triangulated Irregular Network (TIN) surfaces.
```

Comment:
I don't think we currently support all these types yet.

Reply:
I agree. @prantogg Can you explain why you put these here?

Reply:
Yes this is a documentation mistake. Fixed